Applications/Spire/Include/Spire/Spire/RealSpinBoxModel.hpp

```diff
@@ -0,0 +1,126 @@
+#ifndef SPIRE_REAL_SPIN_BOX_MODEL_HPP
+#define SPIRE_REAL_SPIN_BOX_MODEL_HPP
+#include <QLocale>
+#include "Spire/Spire/SpinBoxModel.hpp"
+#include "Spire/Ui/RealSpinBox.hpp"
+
+namespace Spire {
+
+  //! Represents a RealSpinBox's data model.
+  class RealSpinBoxModel {
+    public:
+
```

Comment:
Remove vertical whitespace.

Reply:
Should the destructor be right below the public keyword? I'm only seeing a single space before and after the destructor.

Reply:
Yep.